lighthouse-core/test/audits/manifest-short-name-length-test.js

```diff
@@ -66,6 +67,8 @@ describe('Manifest: short_name_length audit', () => {
     artifacts.Manifest = manifestParser(manifestSrc, EXAMPLE_MANIFEST_URL, EXAMPLE_DOC_URL);
     return ManifestShortNameLengthAudit.audit(artifacts).then(result => {
       assert.equal(result.rawValue, false);
+      assert.ok(result.explanation.includes('without truncation'), result.explanation);
+      assert.ok(!result.explanation.includes('Manifest does not have `short_name`'));
```

Comment:
delete this line as it's not a possibility any longer? Want to assert not not applicable, though?